docs/data/query.md

```diff
@@ -110,50 +110,41 @@ You do not need to perform any additional configuration when [querying data in t
    viam data database configure --org-id=<YOUR-ORGANIZATION-ID> --password=<NEW-DBUSER-PASSWORD>
    ```
 
-   This command configures a new database user for your org for use with data query.
+   This command configures a new database user for your org for use with data query, and sets the password.
+   If you have already run this command before, this command instead updates the password to the new value you set.
 
-1. Determine the hostname for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
+1. Determine the connection URI for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
 
    ```sh {class="line-numbers linkable-line-numbers"}
    viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
    ```
 
-   This command returns the `hostname` (including database name) to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+   This command returns the hostname (including database name) and connection URI to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+   The connection URI includes everything you need to connect to your organization's MongoDB Atlas Data Federation instance from a valid MQL-compatible client, except for your configured database password.
```

Comment:
[opt] "everything you need to connect" seems a bit vague? I don't know the topic well enough to provide a good suggestion, so feel free to disregard but maybe rephrase this along the lines of 
```suggestion
   The connection URI includes the identification information (hostname, database name, and username) you need to connect to your organization's MongoDB Atlas Data Federation instance from a valid MQL-compatible client, except for your configured database password.
```
or, since "uniform resource **identifier**" already implies identification, maybe this:
```suggestion
   In addition to the connection URI, to connect to your organization's MongoDB Atlas Data Federation instance from a valid MQL-compatible client you need your configured database password.
```
```suggestion
   To connect to your organization's MongoDB Atlas Data Federation instance from a valid MQL-compatible client, you need to provide both the URI and your configured database password.
```

Reply:
'identifier' in this case only refers to identifying the network resource to connect to, and doesn't infer user identification (even though it also includes the username in the connection string). You can user a URI that doesn't include a UN or PW (if you provide it elsewhere, like with a SQL client that has separate user auth fields from DB connection string field.

I like your first suggestion, which helps to clarify that the connection URI contains everything we need to connect, including the other returned data object `hostname`, which itself is only part of what we'd need to connect. Thanks!